cmd/client/command/common.go

```diff
@@ -145,20 +141,22 @@ func handleRequest(httpMethod string, url string, yamlBody []byte, cmd *cobra.Co
 		}
 	}
 
+	p := HTTPProtocal
+	if CommandlineGlobalFlags.ForceTLS {
+		p = HTTPSProtocal
+	}
 	tr := http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: CommandlineGlobalFlags.InsecureSkipVerify},
 	}
 	client := &http.Client{Transport: &tr}
-	resp, body := doRequest(httpMethod, url, jsonBody, client, cmd)
+	resp, body := doRequest(httpMethod, p+url, jsonBody, client, cmd)
 
-	if resp.StatusCode == 400 && strings.Contains(string(body), "Client sent an HTTP request to an HTTPS server") {
-		fmt.Println("Warning: upgraded to HTTPS, it's better to turn on option --force-tls")
-		url = strings.ReplaceAll(url, HTTPProtocal, HTTPSProtocal)
-		resp, body = doRequest(httpMethod, url, jsonBody, client, cmd)
+	msg := string(body)
+	if resp.StatusCode == 400 && strings.Contains(msg, "HTTP") && strings.Contains(msg, "HTTPS") {
```

Comment:
`HTTPS` contains `HTTP`

Reply:
you are right, we can use " HTTP " with spaces or just check HTTPS, which do you prefer?

Reply:
if resp.StatusCode == StatusBadRequest  && !CommandlineGlobalFlags.ForceTLS && strings.Contains(strings.ToUpper(msg), "HTTPS") {